src/table/cells/selections.jsx

```diff
@@ -9,23 +8,30 @@ const useStyles = makeStyles({
     background: 'repeating-linear-gradient(-45deg, #fafafa, #fafafa 2px, #eee 2.5px, #fafafa 3px, #fafafa 5px)',
   },
   selected: {
-    color: '#fff',
+    color: '#fff !important',
```

Comment:
Hoping it wouldn't come to this, but it maybe there is no other way around it. Since there will be another layer of styling soon, I think it is fine for now, then maybe will be refactored